patterns/legacy/exim

```diff
@@ -11,7 +11,7 @@ EXIM_MSG_SIZE (S=%{NUMBER:exim_msg_size})
 EXIM_HEADER_ID (id=%{NOTSPACE:exim_header_id})
 EXIM_SUBJECT (T=%{QS:exim_subject})
 
-EXIM_UNKNOWN_FIELD (?:[A-Za-z]{1,4}=%{NOTSPACE})
+EXIM_UNKNOWN_FIELD (?:[A-Za-z0-9]{1,4}=%{NOTSPACE})
```

Comment:
good catch. It looks like the 2013 release of [Exim 4.82](https://git.exim.org/exim.git/blob/exim-4_82:/doc/doc-txt/NewStuff#l129) broke our ability to capture unknown fields with the introduction of `M8S` prefix 🤦🏼